public/_locales/nl/messages.json

```diff
@@ -0,0 +1,298 @@
+{
+    "appName": {
+        "message": "Niet storen s.v.p."
+    },
+    "appDesc": {
+        "message": "Laat de wereld je niet afleiden."
+    },
+    "blacklist": {
+        "message": "Zwarte lijst"
+    },
+    "whitelist": {
+        "message": "Witte lijst"
+    },
+    "combined": {
+        "message": "Gecombineerd"
+    },
+    "combinedDescription": {
+        "message": "Combineert de zwarte en witte lijst door websites op de witte lijst te negeren en websites op de zwarte lijst te blokkeren"
+    },
+    "whitelistDescription": {
+        "message": "Schakel de witte lijst in om alleen de volgende websites te bezoeken:"
+    },
+    "blacklistDescription": {
+        "message": "Schakel de zwarte lijst in om de volgende websites niet te bezoeken:"
+    },
+    "websiteUrlPlaceholder": {
+        "message": "Voer de volledige website-url in"
+    },
+    "blockingMessage": {
+        "message": "Bericht"
+    },
+    "defaultBlockingMessage": {
+        "message": "Waar denk je naartoe te gaan?"
+    },
+    "displayBlankPage": {
+        "message": "Blanco pagina tonen"
+    },
+    "settings": {
+        "message": "Instellingen"
+    },
+    "logs": {
+        "message": "Logboek"
+    },
+    "donate": {
+        "message": "Doneren"
+    },
+    "help": {
+        "message": "Hulp nodig?"
+    },
+    "about": {
+        "message": "Over"    },
```

Comment:
Can you fix the closing bracket position here please? 😅

Reply:
Oops, fixed!